.github/workflows/release.yml

```diff
@@ -96,8 +96,8 @@ jobs:
       - name: Build with Maven
         run: |
           cd coordinator/
-          mvn -B -ntp versions:set -DremoveSnapshot versions:commit
-          mvn -P dse -B -ntp -q -ff clean package -DskipTests
+          ./mvnw -B -ntp versions:set -DremoveSnapshot versions:commit
```

Comment:
Was not using Maven wrapped here for some reason: should (was used in other places) -- not the root cause.